samples/Common.c

```diff
@@ -554,10 +554,16 @@ STATUS createSampleStreamingSession(PSampleConfiguration pSampleConfiguration, P
 
     ATOMIC_STORE_BOOL(&pSampleStreamingSession->terminateFlag, FALSE);
     ATOMIC_STORE_BOOL(&pSampleStreamingSession->candidateGatheringDone, FALSE);
-
+    MEMSET(&pSampleStreamingSession->twccMetadata, 0x00, SIZEOF(TwccMetadata));
```

Comment:
We don't need this right I see that in side `createSampleStreamingSession` we call:

```
    pSampleStreamingSession = (PSampleStreamingSession) MEMCALLOC(1, SIZEOF(SampleStreamingSession));
```

And since `enableTwcc` is a struct member (non-pointer) it would be guaranteed to set to zero.

Reply:
Good catch! Removed.